medicai/layers/resize.py

```
@@ -0,0 +1,41 @@
+from medicai.utils import resize_volumes
+
+
+class SpatialResize(layers.Layer):
+    def __init__(self, target_shape, interpolation="nearest", **kwargs):
+        super().__init__(**kwargs)
+
+        if interpolation not in ("nearest", "bilinear", "trilinear"):
+            raise ValueError(
+                f"Interpolation must be one of ('nearest', 'bilinear', 'trilinear') "
+                "Got {interpolation}"
+            )
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)